spec/domain/export/pdf/invoice_spec.rb

```diff
@@ -469,6 +492,10 @@ def build_invoice(**attrs)
         [346, 155, "Musterweg 2"],
         [346, 144, "8000 Alt Tylerland"]
       ]
+
+      text_with_position.each_with_index do |l, i|
+        expect(l).to eq(invoice_text[i])
+      end
```

Comment:
Schöner Helper. Da dieser jetzt alleine in dieser Spec öfter mal gebraucht wird: kann man hier vllt einen Matcher bauen?